percona/controllers/replica/replica.go

```diff
@@ -307,3 +374,32 @@ func UpdateLabels(cl *crv1.PerconaPGCluster, deployment *appsv1.Deployment) {
 
 	return
 }
+
+func updateDeployment(clientset kubeapi.Interface, replica *crv1.Pgreplica) error {
+	ctx := context.TODO()
+	deployment, err := clientset.AppsV1().Deployments(replica.Namespace).Get(ctx,
+		replica.Name, metav1.GetOptions{})
+	if err != nil {
+		return errors.Wrapf(err, "could not find deployment for pgreplica %s", replica.Name)
+
+	}
+	cl, err := clientset.CrunchydataV1().PerconaPGClusters(replica.Namespace).Get(ctx, replica.Spec.ClusterName, metav1.GetOptions{})
+	if err != nil {
+		return errors.Wrap(err, "get perconapgcluster resource")
+	}
+	updateAnnotations(cl, deployment)
+	updateLabels(cl, deployment)
+	err = pmm.AddOrRemovePMMSidecar(cl, replica.Spec.ClusterName, deployment)
+	if err != nil {
+		return errors.Wrap(err, "add or remove pmm sidecar: %s")
+	}
+	err = updateResources(cl, deployment)
+	if err != nil {
+		return errors.Wrap(err, "update replica resources resource: %s")
```

Comment:
"update replica resources" is enough

Reply:
https://github.com/percona/percona-postgresql-operator/pull/71/commits/a8cbb68cc0989a1d752f02faecd671470266675a